### PR TITLE
moving the test guardian code to live

### PR DESCRIPTION
**Pull request by TimPickup**
Had to re-do Kes part due to conflict.

All tested now in the Kes Test area - now moving to UK.

As discussed previously but didnt work -

This will now consume the amulet on entry.

It also restricts the area to only allow one player at a time as a personal challenge.

Guardian throws fairly frequent WWs but they are just about balmable for an appropriate level player. All in all its not supposed to be a very hard challenge as they have already "paid" its more of a check that a very low level isn't being gifted a hummer.

I have aso removed all the old test code from Kes

### CODE REVIEW

**Comment by TimPickup** [comment]
any questions on this ?

**Review by Chilly021 — APPROVED** [review]

File: Segments/Underkingdom.mapproj
Comment (TimPickup) [reply Segments/Underkingdom.mapproj:102225]: I think I was being cautious or something. probably not really needed. Cant hurt though right?

**Comment by sproatba** [comment]
I was waiting to test it before approving, should be able to now that the portal is fixed

**Comment by TimPickup** [comment]
@sproatba are you aware you need to have a balm in hand to enter the room.. or use the test secret door.

**Comment by sproatba** [comment]
yeah, the surface portal got reverted to the 0,0 version that didn't work

**Comment by sproatba** [comment]
portal in seems to work and consume the balm.  It doesn't prevent me from entering when the guardian is dead though.  Need to find a helper to test the only one can enter part.  Resetting to idle is working and the ability to move and attack when you step on his hex works.

**Comment by TimPickup** [comment]
I would imagine its finding the guardian then but hes marked as dead.. so this line
if( hummingGuardian == null )

would also need to check if || hummingGuardian.dead ? or whatever the property is - i dont recall off the top of my head

**Comment by Chilly021** [comment]
@TimPickup  if you don't mind resolving the conflict, Kes in a state where we are ready for your change. Sorry for the slow down. I spoke with @sproatba  yesterday and confirmed this looks good. As mentioned we can create an interim patch to take care of people getting their amulet consumed without Guardian there.

**Review by sproatba — APPROVED** [review]

**Comment by TimPickup** [comment]
Hi @Chilly021 

I'm not sure there is a lot of point at the moment me resolving the conflicts as there are at elast 3 other Kes PRs out there so another conflict is for sure going to happen and I'm not sure who else is going to approve this PR if I did resolve the conflict.

I'm also not aware how I resolve this conflict. It doesnt let be do it here as I dont have write access and my Git skills are clearly not good enough. Would I pull the current master changes into my personal branch and therefore get the conflict there.. resolve it there and then submit a new update to my branch ? I think basically it would need me to re do the kes changes as i dont think it merges well at all

The update could just be put up without changing Kes as the Kes update is just removing the test code

**Comment by Chilly021** [comment]
Handled in two separate PRS. 
https://github.com/jkachhad/Stormhalter/pull/1694
https://github.com/jkachhad/Stormhalter/pull/1693

**Pull request closed without merging**
